js/src/vercel.ts

```diff
@@ -124,9 +124,39 @@ function convertCoreToSmith(
         }
 
         if (part.type === "image") {
+          let imageUrl = part.image;
+          if (typeof imageUrl !== "string") {
+            let uint8Array;
+            if (
+              typeof imageUrl === "object" &&
+              "type" in imageUrl &&
+              "data" in imageUrl
+            ) {
+              // Typing is wrong here if a buffer is passed in
+              uint8Array = new Uint8Array(imageUrl.data as Uint8Array);
+            } else if (
+              typeof imageUrl === "object" &&
+              Object.keys(imageUrl).every((key) => !isNaN(Number(key)))
```

Comment:
Didn't realize this was a popular format

Reply:
It's not, something is getting badly converted/serialized somewhere in the OTEL bowels